Core/HyBe.Application/Features/Bot/Queries/GetByIdBot/GetByIdBotQuery.cs

```diff
@@ -0,0 +1,19 @@
+﻿using HyBe.Domain.Contracts.Bots;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HyBe.Application.Features.Bot.Queries.GetByIdBot
+{
+    public class GetByIdBotQuery
+    {
+        public GetByIdBotRequest Request { get; set; }
+
```

Comment:
Result sınıfına erişmek için kalıtım ekleyelim 10.satıra. 
`: IRequest<IResult> `